### DISCUSSION TITLE
aiuto @manuelbarberi

### DISCUSSION THREAD

**Discussion by nicholas20220** [iisgiua/giuaschool#312] [Richieste di aiuto]
ciao, ti ho scritto una mail dove ti ho spiegato il problema. fammi sapere se si puo' risolvere grazie.

**Comment by manuelbarberi** [comment]
Ho risposto alla mail… può chiudere la discussione su GitHub

**Comment by nicholas20220** [comment]
Ok grazie

**Comment by nicholas20220** [comment]
Buonasera signor Barberi,
Le chiedo la cortesia di aiutarmi ad aggiornare il registro alla versione 1.45 e passare il registro all'anno successivo in quando non sono competente nel campo.
Ringrazio Anticipatamente

**Comment by trinko** [comment]
Questa non è una segnalazione di errore.
La sposto nelle richieste di aiuto.
